appinventor/buildserver/src/com/google/appinventor/buildserver/resources/runtime.scm

```diff
@@ -1406,10 +1420,16 @@
    ((number? arg) arg)
    ((string? arg)
     (or (padded-string->number arg) *non-coercible-value*))
+   ((enum? arg)
+    (let ((val (arg:getValue)))
+      (if (number? val)
+        val
+        *non-coercible-value*)))
    (else *non-coercible-value*)))
 
 (define (coerce-to-key arg)
   (cond
+   ;;; TODO: I don't understand why these values have to be coerced.
```

Comment:
I don't understand this either!

Reply:
So... do you think I should remove the coersion? Or should I remove the TODO?

Personally I like living dangerously >:D

Reply:
Based on the conversation from our Aug 6 2002 meeting, I recommend leaving a longer comment (1) changing "I" to "Beka" (or something else indicating it's you) or even "Beka and Lyn" and (2) explaining in more detail the nature of the confusion.